blazor/gantt-chart/clipboard.md

```diff
@@ -0,0 +1,495 @@
+---
+layout: post
+title: Clipboard in Blazor Gantt Chart Component | Syncfusion
+description: Checkout and learn here all about clipboard in Syncfusion Blazor Gantt Chart component and much more details.
+platform: Blazor
+control: Gantt Chart
+documentation: ug
+---
+
+# Clipboard in Blazor Gantt Chart Component
+
+The clipboard provides an option to copy selected rows or cells data into the clipboard.
+
+The following list of keyboard shortcuts is supported in the Gantt Chart to copy selected rows or cells data into the clipboard.
+
+Interaction keys |Description
+-----|-----
+<kbd>Ctrl + C</kbd> |Copy selected rows or cells data into clipboard.
+<kbd>Ctrl + Shift + H</kbd> |Copy selected rows or cells data with header into clipboard.
```

Comment:
No need to show this header shortcut

Reply:
Instead explain about CopyHIerarchyMode and clipboard events are missing to cover in UG

Reply:
Explained about  CopyHierarchyMode  here.
https://github.com/syncfusion-content/blazor-docs/pull/1764/files#diff-a16fa1d8ba85b5c925bb17ce8f5889a8d2eddbaeed7cfb2a7a996bfdb19620b9R138

Mentioned about event in below code example.
https://github.com/syncfusion-content/blazor-docs/pull/1764/files#diff-a16fa1d8ba85b5c925bb17ce8f5889a8d2eddbaeed7cfb2a7a996bfdb19620b9R255